lab/13-npm/classes.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+class Person {
+    constructor(cnp){
+        this.cnp = cnp;
+    }
+};
+
+class Customer extends Person {
+    constructor (cnp, money){
+        super(cnp);
+        this.money = money;
+        this.shoppingCart = [];
+    }
+
+    addToCart(product, quantity, available_products){
```

Comment:
As opposed to Python and C where `snake_case` is preferred, in JavaScript `camelCase` is preferred.